modules/microshift-manifests-removal-scenario-rpm.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * microshift//running_applications/microshift-deleting-resource-manifests.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-manifests-removal-scenario-rpm_{context}"]
+== Manifest removal scenario for RPM systems
+Use the following procedure in the data removal scenario for RPM systems to completely delete the resource defined in the kustomize manifests.
+
+.Procedure
+
+. Identify the manifest that needs to be placed in the `delete` subdirectories.
+. Create the `delete` subdirectory in which the manifest will be placed by running the following command:
++
+[source,terminal]
+----
+sudo mkdir -p _<path_of_deletedir>_ <1>
+----
+<1> Replace `_<path_of_deletedir>_` with the path of delete subdirectory such as `/etc/microshift/manifests.d/delete` or `/etc/microshift/manifests/delete/`.
+. Move the manifest file into one of the `delete` subdirectories under the configured manifests directory by running the following command:
++
+[source,terminal]
+----
+[sudo] mv _<path_of_manifests>_ _<path_of_deletedir>_ <1>
+----
+<1> Replace `_<path_of_manifests>_` with the path of manifest that has the manifest to be deleted such as `/etc/microshift/manifests.d/` and replace `_<path_of_deletedir>_` with the path of delete subdirectory such as `/etc/microshift/manifests.d/delete` or `/usr/lib/microshift/manifests/delete/`.
+. Restart {microshift-short} by running the following command:
++
```

Comment:
Important note:
- `manifest` dir is basically a manifest in itself, i.e. kustomization.yaml must be directly inside it
- `manifests.d` dir holds subdirs with manifests, like: `manifests.d/001-multus/kustomization.yaml`, `manifests.d/002-olm/kustomization.yaml`, etc.

So it means that in the former case, they'll basically move the whole dir into a subdir, and for latter they'll move the actual manifest dir.

For example:
- `/usr/lib/microshift/manifests/{kustomization.yaml,some-object.yaml}` -> `/usr/lib/microshift/manifests/delete/{kustomization.yaml,some-object.yaml}`
- `/usr/lib/microshift/manifests.d/010-something/{kustomization.yaml,some-object.yaml}` -> `/usr/lib/microshift/manifests.d/delete/010-something/{kustomization.yaml,some-object.yaml}`
- `/etc/microshift/manifests/{kustomization.yaml,some-object.yaml}` -> `/etc/microshift/manifests/delete/{kustomization.yaml,some-object.yaml}`
- `/etc/microshift/manifests.d/010-something/{kustomization.yaml,some-object.yaml}` -> `/etc/microshift/manifests.d/delete/010-something/{kustomization.yaml,some-object.yaml}`


```suggestion
sudo mkdir -p _<path_of_deletedir>_ <1>
----
<1> Replace `_<path_of_deletedir>_` with the path of delete subdirectory such as `/etc/microshift/manifests.d/delete`, `/etc/microshift/manifests/delete/`, `/usr/lib/microshift/manifests.d/delete`, or `/usr/lib/microshift/manifests/delete`.
. Move the manifest file into one of the `delete` subdirectories under the configured manifests directory by running the following command:
+
[source,terminal]
----
[sudo] mv _<path_of_manifests>_ _<path_of_deletedir>_ <1>
----
<1> Replace `_<path_of_manifests>_` with the path of manifest that has the manifest to be deleted such as `/etc/microshift/manifests.d/010-SOME-MANIFEST` and replace `_<path_of_deletedir>_` with the path of delete subdirectory such as `/etc/microshift/manifests.d/delete`, `/etc/microshift/manifests/delete/`, `/usr/lib/microshift/manifests.d/delete`, or `/usr/lib/microshift/manifests/delete`.
. Restart {microshift-short} by running the following command:
+
```

Reply:
done